src/theme/styled.ts

```diff
@@ -1,5 +1,3 @@
-import { FlattenSimpleInterpolation, ThemedCssFunction } from 'styled-components'
```

Comment:
this fix doesn't need to be in here at all but it does fix the broken unit tests when testing locally. Jest is looking for styled.d and breaks. this entire file neither needs those imports (styled) nor does it need to be a TS `declaration` file. anyways can always revert later but this allows u guys to test locally via `yarn test`